ansible_collections/arista/avd/plugins/action/verify_requirements.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import importlib.metadata
+import re
+import sys
+
+from ansible.plugins.action import ActionBase
+from ansible.errors import AnsibleActionFail
+from ansible.utils.display import Display
+
+HAS_PIP = True
+try:
+    from pip._vendor.packaging.specifiers import SpecifierSet
```

Comment:
Please update requirement.txt to get pip in a min version that has this class

Reply:
Removed pip for packaging